dist/index.html

```diff
@@ -124,8 +124,70 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
       Peoples Feedback
     </section>
 
-    <section class="work-style">
-      Work Style
+    <section class="work-style text-c600">
+      <div class="container">
+        <h2 class="text-center text-c100 font-bold">
+          How We <span class="text-c200 font-hairline underline">Work</span>
+        </h2>
+
+        <div class="showcase-row items-center">
+          <div class="text-center">
+            <div class="border-solid border-6 rounded-full border-c200">
```

Comment:
and i think you can use the border at the parent div instead of the photo itself
and give the two specific width and height
i think it will be better .. try it